tests/authentication/client/test_client_company.py

```diff
@@ -201,6 +201,37 @@ def test_get_client_company(self, mock_email):
                       str(response.data))
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    def test_get_clients_list_unauthenticated(self):
+        """Get clients."""
+        self.client.post(
+            self.client_url, self.valid_client_data, format="json")
+        response = self.client.get(
+            self.clients_list_url, format="json")
+        self.assertIn("Please log in to proceed.",
+                      str(response.data))
+
+    def test_get_clients_list(self):
+        """Get clients."""
+        self.set_token()
+        self.client.post(
+            self.client_url, self.valid_client_data, format="json")
+        response = self.client.get(
+            self.clients_list_url, format="json")
+        self.assertIn("You have retrieved all clients",
+                      str(response.data))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_get_empty_clients_list(self):
+        """Retrieve clients list when no company is created"""
+        self.set_token()
+        from authentication.models import Client
```

Comment:
Hey @Rhytah, it has been team convention to have all module imports at the top of the file, you may as well adopt that.

Reply:
@CryceTruly thanks for pointing this out, I have changed to match your request